### DISCUSSION TITLE
Copy files from repository before mounting storage...

### DISCUSSION THREAD

**Discussion by getschools** [dokku/dokku#5567] [Q&A]
I am deploying a flat-file CMS (Statamic), and I need the `content` directory to have persistent storage, which is easy enough. My next challenge is that there are some files inside the `content` folder that I would like to persist from the git repository. Is there a way to copy the files from the repository into the proper storage folder before deployment?

**Comment by josegonzalez** [comment]
This isn't possible. Consider using separate directories for uploaded content vs statically built artifacts in your app.
